xsh/Makefile.PL

```diff
@@ -9,6 +9,7 @@ $| = 1;
 WriteMakefile(
     'NAME'      => 'XML::XSH2',
     'VERSION_FROM' => 'lib/XML/XSH2.pm',
+    'LICENSE'   => 'gpl_2',
```

Comment:
Is dual licensing possible in Makefile.PL? According to the docs, `artistic_1` should be included, too. [CPAN::Meta::Spec](https://p3rl.org/CPAN::Meta::Spec) seems to allow

    license => [qw[ gpl_2 artistic_1 ]]

but I'm not sure a Makefile.PL can handle that.